### ISSUE TITLE
It does not work Edit: Yes it does, my bad

### ISSUE THREAD

**Issue by I-keep-trying** [tomhicks/themera#5]
I clicked on 'export' and all I got is this; 
```js
{
  "colors": {}
}
```
I clicked on everything but nothing ever happens.

**Comment by archae0pteryx** [comment]
Yup. Can confirm i have the same issue. Too bad too... seems like a useful thing.

**Comment by tomhicks** [comment]
If you haven't modified any colors that's what you'll see. It only outputs what you need to update from the default theme.

**Comment by I-keep-trying** [comment]
OK I see now. Thanks for clearing that up, sorry for my confusion.

Just one opinion, fwiw: What if somebody wants those default colors? Would it not make more sense, if someone clicks "Export" without changing anything, to export the defaults instead of nothing? That's what I initially expected. 

I'm not a design person. I have no idea, honestly, how to pick my own colors in any sort of way like a professional trained designer would. So when I looked at your default colors, I was like, Oh those look nice, lemme just use those. (Those colors are not the Chakra defaults, are they?) That's where I'm coming from. 

So, you did _too_ good of a job on your UI, perhaps! 😊
